eng/templates/stages/deploy.yaml

```diff
@@ -54,7 +54,7 @@ stages:
         useGlobalJson: true
 
     - task: AzurePowerShell@3
-      displayName: eng\firewall.ps1 -Add
+      displayName: /eng/firewall.ps1 -Add
```

Comment:
Is this a "good" Display Name? For me, it looks like a command

Reply:
Oh, it's a display name.. Yeah I can fix that

Reply:
Plus, `displayName: /eng/firewall.ps1 -Remove`